sapi/phpdbg/tests/bug73927.phpt

```diff
@@ -0,0 +1,52 @@
+--TEST--
+Bug #73927 (phpdbg fails with windows error prompt at "watch array")
+--PHPDBG--
+b 19
+r
+c
+w $value
+w $lower[]
+q
+--EXPECTF--
+[Successful compilation of %s]
+prompt> [Breakpoint #0 added at %s:%d]
+prompt> [Breakpoint #0 at %s:%d, hits: 1]
+>00019:     if ($value < 100) {
+ 00020:         $lower[] = $value;
+ 00021:     } else {
+prompt> [Breakpoint #0 at %s:%d, hits: 2]
+>00019:     if ($value < 100) {
+ 00020:         $lower[] = $value;
+ 00021:     } else {
+prompt> [Added watchpoint #0 for $value]
+prompt> [Added watchpoint #1 for $lower[0]]
+prompt> [$lower[0] has been removed, removing watchpoint]
+[$lower[0] has been removed, removing watchpoint]
```

Comment:
Wondering why this message appears twice...

Reply:
Me either. I doesn't happen in interactive mode. Maybe @bwoebi has an idea?

Reply:
```
(gdb) p (char*)phpdbg_globals.watch_recreation->arData[0]->key->val
$8 = 0x1084051f8 "$lower[0]"
(gdb) p (char*)phpdbg_globals.watch_recreation->arData[1]->key->val
$9 = 0x108405108 "$lower[]"
(gdb) p *(phpdbg_watch_element*)phpdbg_globals.watch_recreation->arData[1]->val.value.ptr
$11 = {id = 138956800, watch = 0x10847b0c0, flags = 16 '\020', child = 0x1084852a0, parent = 0x0, ...
```

The latter watch is an implicit (element->flags & PHPDBG_WATCH_IMPLICIT holds) one (i.e. for $lower[0] to be watched, we need a watchpoint on the hashtable).
I think it should be silenced if `child->id` is not present anymore in `PHPDBG_G(watch_elements)`.

Reply:
Ah, thanks a lot!